common/util/autoware_point_types/include/autoware_point_types/types.hpp

```diff
@@ -0,0 +1,90 @@
+// Copyright 2021 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef AUTOWARE_POINT_TYPES__AUTOWARE_POINT_TYPES_HPP_
+#define AUTOWARE_POINT_TYPES__AUTOWARE_POINT_TYPES_HPP_
+
+#include <point_cloud_msg_wrapper/point_cloud_msg_wrapper.hpp>
+
+#include <cmath>
+#include <limits>
+#include <tuple>
+
+namespace
+{
```

Comment:
Please follow this rule.
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#sf21-dont-use-an-unnamed-anonymous-namespace-in-a-header

Reply:
I fixed in https://github.com/tier4/AutowareArchitectureProposal.iv/pull/784/commits/084d75ac4bbd207ff4a7631d453c008f48ef1e14